packages/date-picker/src/vaadin-date-picker-helper.js

```diff
@@ -117,3 +117,29 @@ export function dateAfterXMonths(months) {
   result.setMonth(parseInt(months) + today.getMonth());
   return result;
 }
+
+/**
+ * Calculate the year of the date based on the provided reference date.
+ * Gets a two-digit year and returns a full year.
+ * @param {!Date} referenceDate The date to act as basis in the calculation
+ * @param {!Number} year Should be in the range of [0, 99]
+ * @param {Number} month
+ * @param {Number} day
+ * @return {!Number} Adjusted year value
+ */
+export function getAdjustedYear(referenceDate, year, month, day) {
```

Comment:
Consider adding fallback values for optional parameters:

```suggestion
export function getAdjustedYear(referenceDate, year, month = 0, day = 1) {
```